examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -9,7 +9,6 @@ angular.module('cloudberry.common', [])
       normalizationUpscaleFactor: 1000 * 1000,
       normalizationUpscaleText: "/M",
       sentimentUpperBound: 4,
-      cacheThreshold: config.cacheThreshold,
```

Comment:
why this was removed?

Reply:
This line and line 105 are likely due to improper rebasing.